sessions/backend-basics/routes/package.json

```
@@ -0,0 +1,23 @@
+{
+  "name": "backend-basics_routes",
+  "version": "0.0.0-unreleased",
+  "description": "Backend Basics: Routes",
+  "main": "index.js",
+  "type": "module",
+  "scripts": {
+    "start": "node index.js",
+    "lint": "npx eslint .",
+    "test": "NODE_OPTIONS=--experimental-vm-modules npx jest --watchAll"
```

This would be the default output of the `test` script of no tests are provided
```suggestion
    "test": "echo \"Error: no test specified\" && exit 1"
```